webtraversallibrary/webdrivers.py

```diff
@@ -98,6 +98,37 @@ def _add_script(driver: WebDriver, script_path: str):
     return driver
 
 
+def _setup_firefox(config: Config, _profile_path: Path = None) -> WebDriver:
+    firefox_profile = webdriver.FirefoxProfile()
+    firefox_options = webdriver.FirefoxOptions()
+    browser = config.browser
+
+    # Disable security
+    firefox_options.add_argument("-safe-mode")
+
+    # Force opening in new tabs
+    # firefox_options.set_preference("browser.link.open_newwindow", 3)
+    # firefox_options.set_preference("browser.link.open_newwindow.restriction", 0)
+
```

Comment:
[Found how to disable audio in Firefox as well](https://stackoverflow.com/questions/12158366/disable-sound-from-firefox-selenium-webdriver-programming), but couldn't find how to disable scrollbars or the other stuff that Chrome does (for now).

```suggestion

# Disable unused/annoying features
firefox_options.setPreference("media.volume_scale", "0.0")

```